vscode-dotnet-runtime-library/src/test/unit/WebRequestWorker.test.ts

```diff
@@ -72,13 +72,13 @@ suite('WebRequestWorker Unit Tests', () => {
 
     test('Web Requests Memoized on Repeated Installs', async () => {
         const [eventStream, context] = getTestContext();
-        const webWorker = new MockWebRequestWorker(context, eventStream, '', 'MockKey');
+        const webWorker = new MockWebRequestWorker(context, eventStream,);

```

Comment:
No additional tests for the new JSON method?

Reply:
This method been removed with some of the changes we made over the past month, resolving this comment as those methods have tests ;)